src/app/wishlists/wishlists.ts

```diff
@@ -87,8 +80,8 @@ function isWeaponOrArmorOrGhostMod(plug: DimPlug): boolean {
   if (
     plug.plugItem.itemCategoryHashes?.find(
       (ich) =>
-        ich === INTRINSIC_ITEM_CATEGORY || // intrinsics
-        ich === WEAPON_GAMEPLAY_MODS_CATEGORY || // weapon gameplay socket
+        ich === ItemCategoryHashes.WeaponModsIntrinsic ||
+        ich === ItemCategoryHashes.WeaponModsGameplay ||
         ich === 3851138800 // armor gameplay socket
```

Comment:
this is not in `generated-enums.ts`

Reply:
yep, see https://github.com/DestinyItemManager/DIM/issues/5521